access/src/main/webapp/WEB-INF/jsp/fullRecord/browseView.jsp

```diff
@@ -18,10 +18,70 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
```

Comment:
Hmm, I missed it on a previous PR, but I didn't realize that browseView.jsp was being used in place of having an fullRecord/adminUnit.jsp. This file appears to be the adminUnit full record display at this point, perhaps we should rename it? Or move all of this new stuff to a new adminUnit.jsp and import browseView in it? Collections and folders are both going to need the browse view as well